File: Leetcode/9-Palindrome-Number/PalindromeNumber.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        string str = to_string(x);
        int i=0;
        int j=str.length()-1;
        while (i<j) {
            if (str.at(i) != str.at(j)) {
                return false;
            }
            i++;
            j--;
        }
        return true;
    }
};

File: Leetcode/9-Palindrome-Number/PalindromeNumber.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        s = str(x)
        i = 0
        j = len(s)-1
        while i<j :
            if s[i]!= s[j]:
                return False
            i+=1
            j-=1
        return True
    
